docs/src/prototypes/CopyToClipboard/NotificationProvider.tsx

```diff
@@ -28,9 +35,21 @@ export const NotificationProvider: React.FC = props => {
 
   return (
     <>
-      <Portal open={!!notification}>
-        <Notification>{notification}</Notification>
-      </Portal>
+      {!!targetRef ? (
+        !!notification && (
+          <Tooltip
+            content={notification}
+            pointing={true}
+            open={!!notification}
+            target={targetRef.current}
+            variables={{ primary: true }}
+          />
+        )
+      ) : (
+        <Portal open={!!notification}>
+          (<Notification>{notification}</Notification>)
+        </Portal>
```

Comment:
Condition there is very unclear.

Why we show `Notification` or `Tooltip` based on `targetRef`? Should we introduce an additional `position` or prop?

Reply:
changed the API - now there is a separate prop for tooltip (shorthand) and a separate for ataching the notification (TooltipProps or boolean, but we can change it to just subset of tooltip props)